src/datepickers/datepicker.ts

```diff
@@ -36,8 +36,8 @@ export class NglDatepicker {
   }
 
   firstDayOfWeek = 0;
-  @Input('firstDayOfWeek') set _firstDayOfWeek(firstDayOfWeek: number) {
-    this.firstDayOfWeek = firstDayOfWeek;
+  @Input('firstDayOfWeek') set _firstDayOfWeek(firstDayOfWeek: any) {
+    this.firstDayOfWeek = (typeof firstDayOfWeek) === 'number' ? firstDayOfWeek : parseInt(firstDayOfWeek);
```

Comment:
@sdiprizio Since we expect a number like input, I think it would be ok to always convert to number like this:
```js
this.firstDayOfWeek = +firstDayOfWeek;
```